style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  font-family: 'open', sans-serif;
+}
+
+.box-container {
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  inset: 0;
+  margin: auto;
+}
+
+.MILINDI {
+  width: 10%;
+  padding: 30px;
+  background: rgb(231, 171, 4);
+  border-bottom-left-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.SUV {
+  width: 10%;
+  padding: 30px;
+  background: rgb(7, 96, 131);
+}
+
+.LUXURY {
+  width: 10%;
+  padding: 30px;
+  background: rgb(2, 53, 40);
+  border-bottom-right-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.box-header h3 {
+  text-transform: uppercase;
+  font-size: 28px;
+}
+
+.box-img img {
+  width: 60px;
+}
+
+.box-body p {
+  margin-bottom: 50px;
+}
+
+.box-body a {
+  background: #fff;
+  color: rgb(231, 171, 4);
+  text-decoration: none;
+  padding: 10px 20px;
+  border: 2px solid #fff;
+  border-radius: 50px;
+}
+
+.box-body a:hover {
+  color: #fff;
+  background: rgb(231, 171, 4);
+  border: 2px solid #fff;
+}
+
+/* .box-body a {
+  background: #fff;
+  color: rgb(7, 96, 131);
+  text-decoration: none;
+  padding: 10px 20px;
+  border: 2px solid #fff;
+  border-radius: 50px;
+} */
+
+/* .box-body a:hover {
+  color: #fff;
+  background: rgb(7, 96, 131);
+  border: 2px solid #fff;
+} */
+
+/* .box-body a {
+  background: #fff;
+  color: rgb(2, 53, 40);
+  text-decoration: none;
+  padding: 10px 20px;
+  border: 2px solid #fff;
+  border-radius: 50px;
+} */
+
+/* .box-body a:hover {
```

Comment:
- [ ] `required`
- [ ] As explained earlier kindly use words like `please`, `kindly`, and `I suggest` and other words which can be polite because we don't want to sound rude and discourage the student and be `polite` also use `emoji`